backend/src/tasks/cve.ts

```diff
@@ -99,12 +101,72 @@ const identifyPassiveCVEs = async () => {
   await saveVulnerabilitiesToDb(vulnerabilities, false);
 };
 
+interface NvdFile {
+  CVE_Items: {
+    cve: {
+      CVE_data_meta: {
+        ID: string;
+      };
+      references: {
+        reference_data: {
+          url: string;
+          name: string;
+          refsource: string;
+          tags: string[];
+        }[];
+      };
+      description: {
+        description_data: {
+          lang: string;
+          value: string;
+        }[];
+      };
+    };
+  }[];
+}
+
 // Populate CVE information
 const populateVulnerabilities = async () => {
   const vulnerabilities = await Vulnerability.find({
     needsPopulation: true
   });
+  const vulnerabilitiesMap: { [key: string]: Vulnerability[] } = {};
+  for (const vuln of vulnerabilities) {
+    if (vuln.cve) {
+      if (vulnerabilitiesMap[vuln.cve]) {
+        vulnerabilitiesMap[vuln.cve].push(vuln);
+      } else vulnerabilitiesMap[vuln.cve] = [vuln];
+    }
+  }
   // TODO: Populate info of these vulnerabilities
+  const filenames = await fs.promises.readdir('nvd-dump');
+  for (const file of filenames) {
```

Comment:
Looks good -- can you update the cve test to account for populateVulnerabilities as well?